### PR TITLE
Issue#294 eth faucet

### PR DESCRIPTION
**Pull request by pldespaigne**

### CODE REVIEW

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html [hunk 1]
```diff
@@ -28,7 +28,14 @@ <h1>Michel Michelle</h1>
           <span class="key">Sold Address</span>
           <mat-icon matTooltip="Info about the action" matTooltipPosition="right">info_outline</mat-icon>
         </div>
-        <span class="sold">1 000</span>
+        <ng-container *ngIf="!(isBalanceLoading$ | async); else isLoading">
```
Comment (GrandSchtroumpf) [inline apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html]: Note : This should be a component just after Cannes.

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html [hunk 2]
```diff
@@ -66,10 +73,10 @@ <h1>Buy Tokens</h1>
       <div fxLayout="column">
         <div fxFlex fxLayout="row" fxLayoutGap="20px">
```
Comment (GrandSchtroumpf) [inline apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html]: Are you sure you need fxFlex here and below ?

File: libs/account/src/lib/account-view/account-view.component.html [hunk 3]
```diff
@@ -64,17 +64,14 @@
             Wallet
           </mat-panel-title>
         </mat-expansion-panel-header>
-        <mat-list>
-          <mat-list-item>
+          <mat-card class="mat-elevation-z0">
```
Comment (GrandSchtroumpf) [inline libs/account/src/lib/account-view/account-view.component.html]: Warning it's gonna break my PR.

File: libs/account/src/lib/account-view/account-view.component.scss [hunk 4]
```diff
@@ -25,6 +25,16 @@ mat-spinner {
   margin: 6px 0 6px 12px;
 }
 
+div > button {
+  height: 51px;
+  position: relative;
```
Comment (GrandSchtroumpf) [inline libs/account/src/lib/account-view/account-view.component.scss]: I'm pretty sure you don't need relative here, and botton too.

File: libs/account/src/lib/account-view/account-view.component.scss [hunk 5]
```diff
@@ -25,6 +25,16 @@ mat-spinner {
   margin: 6px 0 6px 12px;
 }
 
+div > button {
+  height: 51px;
```
Comment (GrandSchtroumpf) [inline libs/account/src/lib/account-view/account-view.component.scss]: Can't it really be 50px ? ^^

File: libs/auth/src/lib/+state/auth.service.ts [hunk 6]
```diff
@@ -105,6 +105,12 @@ export class AuthService {
     this.store.update({isBalanceLoading: false});
   }
 
+  public async requestTokens(amount: number) {
+    this.store.update({isBalanceLoading: true});
+    await this.wallet.requestTokens(amount).catch(error => console.error('Request Tokens FAILED because of :',error));
```
Comment (GrandSchtroumpf) [inline libs/auth/src/lib/+state/auth.service.ts]: Do try / catch synthax here.

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html (hunk 1 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html]: #319

File: apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html (hunk 2 above)
Comment (pldespaigne) [reply apps/movie-financing/movie-financing/src/app/explorer/profile/profile.component.html]: ✔️

File: libs/account/src/lib/account-view/account-view.component.scss (hunk 5 above)
Comment (pldespaigne) [reply libs/account/src/lib/account-view/account-view.component.scss]: NO ! 😝 
image

File: libs/account/src/lib/account-view/account-view.component.scss (hunk 4 above)
Comment (pldespaigne) [reply libs/account/src/lib/account-view/account-view.component.scss]: ✔️

File: libs/auth/src/lib/+state/auth.service.ts (hunk 6 above)
Comment (pldespaigne) [reply libs/auth/src/lib/+state/auth.service.ts]: ✔️

File: libs/account/src/lib/account-view/account-view.component.html (hunk 3 above)
Comment (pldespaigne) [reply libs/account/src/lib/account-view/account-view.component.html]: ✔️

File: libs/account/src/lib/account-view/account-view.component.scss (hunk 5 above)
Comment (GrandSchtroumpf) [reply libs/account/src/lib/account-view/account-view.component.scss]: What about  : 
```html
<div class="my-form" fx-layout="row" fx-layout-align="start center">
  <mat-form-field>
    <input />
  </mat-form-field>
  <button mat-button>Buy</mat-button>
</div>
```
```
.my-form {
  box-sizing: border-box;
  height: 50px;
}
.my-form * {
  height: 100%;
}
```
In any case, this is not a good practice to force the height like that.

**Pull request merged**